src/scratch/lunar_lander/README.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Lærerveiledning - Lunar Lander
+level: 4
+language: nb
+---
+
+# Om oppgaven {.activity}
+Lunar Lander ble opprinnelig utviklet på slutten av 1970-tallet. Målet
+med spillet er å lande et romskip på månen. Selve kontrollen av
+romskipet ble videreført i spillet Asteroids som ble utgitt året
+etter.
+
+![](lunar_lander.png)
+
+## Oppgaven passer til: {.check}
+ __Fag__: Kunst og håndtverk, matematikk, programmering.
+__Anbefalte trinn__: 2.-10. trinn.
```

Comment:
vil si at denne er for 5. klasse og oppover (litt dumt når 5/6 kompetansemål er for yngre elever, men den er litt vanskelig)